src/core/agent/agent.ts

```diff
@@ -63,7 +68,7 @@ class Agent {
   static readonly MISSING_DATA_ON_KERIA =
     "Attempted to fetch data by ID on KERIA, but was not found. May indicate stale data records in the local database.";
   static readonly BUFFER_ALLOC_SIZE = 3;
-
+  static readonly APP_PASSSCODE_DEV_MODE = "111111";
```

Comment:
Lets just keep this directly in the function. static variables here should be for the real implementation and not dev mode related things